src/components/Provider.ts

```diff
@@ -34,15 +34,18 @@ type withProviderPrivates = VueConstructor<
 
 function data() {
   const errors: string[] = [];
+  const successes: string[] = [];
```

Comment:
I don't think both `successes` and `resolvedRules` are needed.

Reply:
I just followed the design with errors and failedRules that exists in code. I think it's a good idea to have some consistency.